llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -2331,7 +2331,7 @@ void VPWidenLoadEVLRecipe::execute(VPTransformState &State) {
 #if !defined(NDEBUG) || defined(LLVM_ENABLE_DUMP)
 void VPWidenLoadEVLRecipe::print(raw_ostream &O, const Twine &Indent,
                                  VPSlotTracker &SlotTracker) const {
-  O << Indent << "WIDEN ";
+  O << Indent << "WIDEN-VP ";
```

Comment:
`-VP` may be confusing as VP is used for various VPlan-related classes as well. Simpler to just update `WIDEN-VP` -> `WIDEN ... = vp....`? 

Otherwise VPreductionEVLRecipe also needs updating.

Reply:
done